test/node-disk-fill-test.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-env mocha */
+
+'use strict';
+
+const os = require('os');
+const assert = require('bsert');
+const FullNode = require('../lib/node/fullnode');
+const RAMDisk = require('./util/ramdisk');
+
+describe('Node Disk Fill', function() {
+  this.timeout(30000);
+
+  switch (os.platform()) {
+    case 'darwin':
+    case 'linux':
+      break;
+    default:
+      this.skip();
+  }
+
+  if (!process.env.HSD_TEST_SUDO)
+    this.skip();
+
+  let disk, node;
+
+  beforeEach(async () => {
+    disk = new RAMDisk(1024 * 1024);
+    await disk.open();
+  });
+
+  afterEach(async () => {
+    if (node && node.opened)
+      await node.close();
+
+    if (disk && disk.path)
+      await disk.close();
+  });
+
+  process.on('SIGINT', async () => {
```

Comment:
I am not sure why this is here. Also if we 100% need this, we need to clean up after this test is done, but I don't see reason for this.

Reply:
This was a failsafe for cleaning up the ramdisk when writing the tests locally. We can remove it and the whole ramdisk thing